package.yaml

```diff
@@ -42,6 +42,7 @@ dependencies:
   - optics-core
   - optics-th
   - ansi-terminal
+  - weighted-search
```

Comment:
I did not spot any place where this dependency is used. I assume you've added it for a future work.

Reply:
It's used for its `Weight` type class; but I could also inline that definition and remove the dependency.